src/components/modulesList/Item.js

```diff
@@ -1,37 +1,49 @@
-import React from "react";
+import React, { useContext } from "react";
 import PropTypes from "prop-types";
+import { Icon } from "react-native-elements";
 import { StyleSheet, Text, TouchableOpacity, View } from "react-native";
-import { AntDesign } from "@expo/vector-icons";
 import { Colors, Typography } from "../../constants/styles";
+import { DiagnosisContext } from "../../modules/context/DiagnosisContext";
+
+const ModuleItem = ({ moduleCode, onPress }) => {
+  const { modules } = useContext(DiagnosisContext);
```

Comment:
dla każdego z ModuleItem musimy odpytywać kontekst? nie wystarczy jeśli w ModulesList wyciągniemy wszystkie dane o modułach z kontekstu? od czego to zależy?

Reply:
W sumie mógłbym zrobić tak, że przekazuję te dane w propsach zamiast używać konteksu, ale stwierdziłem, że skoro już go mam to z niego skorzystam i będę miał mniej rzeczy do przekazywania. 
Z tego co wiem to nie ma żadnej różnicy wydajnościowo.